packages/server/graphql/mutations/helpers/sendAccountRemovedToSegment.ts

```diff
@@ -26,14 +26,7 @@ const sendAccountRemovedToSegment = async (
   })
   parabolPayload.data.user.email = email
   parabolPayload.data.user.isRemoved = true
-  segmentIo.track({
-    userId: userIdToDelete,
-    event: 'Account Removed',
-    properties: {
-      reason: validReason,
-      parabolPayload: parabolPayload.data
-    }
-  })
+  analytics.accountRemoved(userIdToDelete, validReason, parabolPayload.data)
```

Comment:
-1 I think the fact that we encode the `email` and `isRemoved` in an object `parabolPayload` should be an implementation detail in `analytics.ts` and we should just call `analytics.accountRemoved(userId, email, reason)` here.

Reply:
We're not using the `parabolPayload` data anywhere, and I don't see a lot of value in capturing this data.  I'm removing it entirely.